chapters/VK_EXT_swapchain_maintenance1/SwapchainPresentFenceInfo.adoc

```diff
@@ -40,9 +40,9 @@ fence must: not signal until all such semaphore payloads have been reset by
 the presentation engine.
 
 The application can: destroy the wait semaphores associated with a given
-presentation operation when the associated fence is signaled, and can:
-destroy the swapchain when the fences associated with all past presentation
-requests have signaled.
+presentation operation when at least one of the associated fences is signaled,
+and can: destroy the swapchain when the fences associated with all past
+presentation requests have signaled.
```

Comment:
Given the ambiguity pointed out by the language changed here, perhaps we should be clearer about the second clause as well. It's all fences associated with all past presentation requests referring to that swapchain.

Reply:
I've included that as well now.